src/Common/Configuration.ts

```diff
@@ -30,15 +29,12 @@ export class Configuration {
 
         nconf.required(Configuration._requiredConfigKeys);
 
-        this.sqlConfig = {
-            database: nconf.get(Configuration._configKeys.databaseName),
-            options: {
-                encrypt: true
-            },
-            password: nconf.get(Configuration._configKeys.databasePassword),
-            server: nconf.get(Configuration._configKeys.databaseServer),
-            user: nconf.get(Configuration._configKeys.databaseUsername)
-        };
-        this.port = process.env.PORT || 3000;
+        this.port = process.env.PORT || "3000";
+
+        const dbName: string = nconf.get(Configuration._configKeys.databaseName);
```

Comment:
I guess this is a step better than exposing all these properties publicly, but it's not exactly what I meant - I meant put the connection string (as a single string) in an environment variable that will be populated by your web app settings and loaded it from there. this makes it easier to load (you have 1 setting instead of 4), makes it easier to migrate to managed identity at some point (no code changes needed) and is natively supported by web apps.

Reply:
Ok

Reply:
Come to think of it, I don't think I can do it at this point because I need to encode the password if it has symbols like %@$: etc...  I can't encode the entire string because it has username:password@server etc...   I'll keep it as it.